pkg/controller/node/ipam/cloud_cidr_allocator.go

```diff
@@ -197,11 +197,21 @@ func (ca *cloudCIDRAllocator) updateCIDRAllocation(nodeName string) error {
 
 	cidrs, err := ca.cloud.AliasRanges(types.NodeName(nodeName))
 	if err != nil {
-		util.RecordNodeStatusChange(ca.recorder, node, "CIDRNotAvailable")
+		node, err = ca.nodeLister.Get(nodeName)
```

Comment:
can we combine the error cases into one block:
```
if err != nil || len(cidrs) == 0 {
  ....
}
```
Looks like the same code

Reply:
Done! The error is different though so it requires an extra if inside.